rootfs-configs.yaml

```diff
@@ -23,6 +25,7 @@ rootfs_configs:
       - libp11-kit0
       - libunistring2
       - sensible-utils
+      - isc-dhcp-client
```

Comment:
This package is added and then removed.  Is this to have it only in the full rootfs image for nfsroot and not in the ramdisk version?

If it's not making the ramdisk much bigger then it could probably be left there, or maybe some files like man pages could be removed to optimize it.  The size of the buster ramdisk is not as critical as the buildroot one, as long as most devices can still load it.